docs/_articles/en/einstein/einstein-example-prompts.md

```diff
@@ -1,23 +1,121 @@
+
+4. **Additional Context**: Any additional information that you want to provide.
 
-| Original Prompt                                         | Improved Prompt                                                                |
-| ------------------------------------------------------- | ------------------------------------------------------------------------------ |
-| Write Apex code that sorts the list of contacts.        | Write Apex code that sorts the list of contacts alphabetically by last name.   |
-| Write a method that adds two numbers.                   | Write a method that adds two numbers and returns the result.                   |
-| Write an Apex method that returns fields for an object. | Write an Apex method that accepts an SObject and returns a list of its fields. |
+For example:
+_Create an Apex batch class named "OpenPositionMatcher" that processes custom objects "Position\_\_c" records created within the past 15 days. The class should match these open positions with suitable candidates based on their skills." Order the results by candidate names in descending order. Make sure the generated code follows security best practices. Separate operators with a new line. The class is used in an LWC component._
+
+The final prompt gives you a response that looks something like this:
+
+```
+public with sharing class OpenPositionMatcher {
+
+    @AuraEnabled(cacheable=true)
+    public static List<Candidate__c> matchSkills(String positionId) {
+        try {
+            Position__c position = [SELECT Id, Skills__c FROM Position__c WHERE Id = :positionId WITH SECURITY_ENFORCED];
+            List<String> skills = position.Skills__c?.split(';');
+
+            if (skills != null && !skills.isEmpty()) {
+                List<Candidate__c> candidates = [
+                    SELECT Id, Name, Skills__c
+                    FROM Candidate__c
+                    WHERE Skills__c IN :skills
+                    WITH SECURITY_ENFORCED
+                    ORDER BY Name DESC
+                ];
+
+                return candidates;
+            }
+        } catch (Exception e) {
+            System.debug(e.getMessage());
+            throw new AuraHandledException(e.getMessage());
+        }
+
+        return new List<Candidate__c>();
+    }
+}
+
+```
+## Example Prompts and Context Descriptions
+<table>
+    <tr>
+      <th>Prompt</th>
+      <th>Context Description</th>
+    </tr>
+    <tr>
+      <td>Create a class named `OpportunityController`. Create a method called `getNumberofOpportunities`. The method takes `AccountId` as a parameter and returns the number of opportunities for that `AccountId`. I’m working on an LWC component. Make sure the SOQL uses security best practices like user mode data operations.</td>
+      <td <ul>
```

Comment:
@sbudhirajadoc missing a `>` on the td on this row.  Might be the root of our table rendering issue.

Reply:
Thanks @gbockus-sf . I'll look into it. It renders correctly in preview so it's hard to know how it will finally render until it's published...I am leaning towards getting rid of the table and conveying the information some other way.